psi4/src/psi4/dlpno/mp2.h

```diff
@@ -130,6 +130,7 @@ class DLPNOMP2 : public Wavefunction {
 
     // Dense analogues of some sparse maps for quick lookup
     std::vector<std::vector<int>> riatom_to_lmos_ext_dense_;
+    std::vector<std::vector<int>> riatom_to_paos_ext_dense_;
```

Comment:
It took me far too long to figure out what this represented. Please make the comments in this part of the header more detailed. This takes an atom index and a PAO index and returns _the index of that PAO within the non-dense version_. It wasn't at all clear to me that the index within the non-dense version was important. To be fair, the old docs are not clear that this is what the SparseMaps store.